src/navigation/pathnavigator.cpp

```diff
@@ -263,11 +257,20 @@ void PathNavigator::startPath() {
         return;
     }
 
+    if (!global::openSpaceEngine->setMode(OpenSpaceEngine::Mode::CameraPath)) {
+        LERROR("Could not start camera path");
+        return; // couldn't switch to camera path mode
+    }
+
     // Always pause the simulation time when flying, to aovid problem with objects
     // moving. However, keep track of whether the time was running before the path
     // was started, so we can reset it on finish
     if (!global::timeManager->isPaused()) {
-        global::timeManager->setPause(true);
```

Comment:
Why switch to the script queue for the pause? Is this for parallel peer? (which I guess if so would make sense)

Reply:
Mainly so that the pause is recorded in the session recording, and synced out to all the machines when for example in a dome setup. But yeah, I guess parallel peer is another use case